README.md

```diff
@@ -1,9 +1,109 @@
+
+There's [work being done (#27)](https://github.com/minimization/content-resolver/issues/27) to change the way build dependencies are resolved (using data from Koji rather than the SRPMs themselves) that will remove the current limitations.
+
+### Reasons of Presence
+
+To make minimisation of a set defined by a view easier, each package gets its detail page that shows what it is present.
+
+This page includes information about what Workloads pull it in and who owns the workload, as well as package-level dependencies, both runtime and build.
+
+### Unwanted Packages
+
+Another way to make minimisation easier is the concept of unwanted packages.
+
+Maintainers can flag certain packages as unwanted. That causes these packages to be highlighted (if they're present in the set), and also listed on dedicated page.
+
+In combination with Reasons of Presence, this helps maintainers identify what dependencies need to be cut in order to remove the unwanted packages.
+
+Current limitations: There's currently only a single level of unwanted packages. But there's [work being done (#28)](https://github.com/minimization/content-resolver/issues/28) to allow additional levels, such as a distinction between "unwanted in the runtime set" vs. "unwanted in buildroot as well".
+
+### Maintainer recommendation
+
+Because views are defined by multiple parties, it's not always clear who should be responsible for owning shared dependencies.
+
+That's why Content Resolver helps identify who pulls each packages into the set "the most" and recommends ownership based on that.
+
+Current limitations: Content Resolver doesn't look at anything else, just the dependencies and who pulled it in. But parties can volunteer to own a package. In that case, Content Resolver should be able to see that and show that instead. There's [work being done (#29)](https://github.com/minimization/content-resolver/issues/29) that will allow maintainers to accept a package, allowing Content Resolver to use this information when recommending owners.
+
+## Running Content Resolver
+
+Content Resolver is designed to be highly available without needing the operator to run critical infrastructure.
+
+The input is consumed from a git repository. This allows to use existing services such as GitHub.
+
+The output is just a set of static HTML pages. This allows it to be served from CDNs using services such as AWS S3.
+
+The only thing that remains is the script that reads the input and regenerates the output. This can be run as a periodic job in various places. What's important is that this script only refreshes the output, meaning that it doesn't need to be kept running for the service to be available. This means upgrades and maintenance won't cause down time.
```

Comment:
```suggestion
Lastly, there is a script that reads the input and regenerates the output. This can be run as a periodic job on any appropriate system. Notably, this script only refreshes the output, meaning that it doesn't need to be kept running for the service to be available and that upgrades and maintenance of the Content Resolver won't result in down-time of the resolved views.
```